demos/iteration/src/test/java/com/datatorrent/demos/iteration/ApplicationTest.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datatorrent.demos.iteration;
+
+
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Test;
+
+import com.datatorrent.api.LocalMode;
+
+
+/**
+ *
+ */
+public class ApplicationTest
+{
+  @Test
+  public void testSomeMethod() throws Exception
+  {
+	  LocalMode lma = LocalMode.newInstance();
+	    Configuration conf =new Configuration(false);
+	    lma.prepareDAG(new Application(), conf);
+	    LocalMode.Controller lc = lma.getController();
+	    lc.run(10000);
```

Comment:
Please modify the test to check for the result instead of waiting for arbitrary time. One way to do this is to write output to file. Run run app async and check file content for expected result. Example: https://github.com/DataTorrent/examples/blob/master/tutorials/hdht/src/test/java/com/example/HDHTAppTest.java#L87